pkg/test/framework/components/redis/redis.yaml

```diff
@@ -146,11 +145,11 @@ spec:
       serviceAccountName: "default"
       containers:
       - name: redis-release
-        image: docker.io/bitnami/redis:4.0.12
-        imagePullPolicy: "Always"
+        # Assumption we use the same registry for istio images and the bitnami images
+        image: "{{ (split '/' .Hub)._0 }}/bitnami/redis:4.0.1"
```

Comment:
My hub, and I suspect others is `gcr.io/istio-release` or `gcr.io/my-project`. Now will this try to pull `gcr.io/bitnami/redis` which doesn't exist?

Reply:
Okay, so we need to implement the logic with a second variable/flag. I would suggest name it `BitnamiHub` since both images are `bitnami` images that are pulled (`redis`/`minideb`).

Reply:
I added the `BitnamiHub`, any pointers why the presubmit step is failing ?